laboratory-servlet-api-integration/src/main/java/com/wixpress/petri/laboratory/CountryResolver.scala

```diff
@@ -0,0 +1,19 @@
+package com.wixpress.petri.laboratory
+
+import javax.servlet.http.HttpServletRequest
+
+object CountryResolver {
+
+  def resolve(request: HttpServletRequest, requestDataSourcesConfiguration: FilterParametersExtractorsConfig): String = {
```

Comment:
requestDataSourcesConfiguration: FilterParametersExtractorsConfig

i guess these 2 names show you're not sure yet what the class should be called.

lets stick to just one everywhere to make future renaming easier :)